cmd/show/dashboard.go

```diff
@@ -0,0 +1,360 @@
+
+	issuesAndForks := lipgloss.JoinVertical(lipgloss.Center, TextContainer.Render(repoIssues), TextContainer.Render(repoForks))
+	sizeAndStars := lipgloss.JoinVertical(lipgloss.Left, TextContainer.Render(repoSize), TextContainer.Render(repoStars))
+
+	repoGeneralSection := lipgloss.JoinHorizontal(lipgloss.Center, issuesAndForks, sizeAndStars)
+	repositoryInfoView := lipgloss.JoinVertical(lipgloss.Center, repoName, repoGeneralSection)
+	frame := SquareBorder.Render(repositoryInfoView)
+
+	return frame
+}
+
+// drawMetrics: view of metrics includes.
+// - new contributors table
+// - alumni contributors table
+func (m *DashboardModel) drawMetrics() string {
+	var newContributorsDisplay, alumniContributorsDisplay string
+
+	switch m.tableView {
+	case newContributorsView:
+		newContributorsDisplay = lipgloss.JoinVertical(lipgloss.Center, TableTitle.Render("🍕 New Contributors"), ActiveStyle.Render(m.newContributorsTable.View()))
+		alumniContributorsDisplay = lipgloss.JoinVertical(lipgloss.Center, TableTitle.Render("🍁 Alumni Contributors"), InactiveStyle.Render(m.alumniContributorsTable.View()))
+	case alumniContributorsView:
+		newContributorsDisplay = lipgloss.JoinVertical(lipgloss.Center, TableTitle.Render("🍕 New Contributors"), InactiveStyle.Render(m.newContributorsTable.View()))
+		alumniContributorsDisplay = lipgloss.JoinVertical(lipgloss.Center, TableTitle.Render("🍁 Alumni Contributors"), ActiveStyle.Render(m.alumniContributorsTable.View()))
+	}
+
+	contributorsMetrics := lipgloss.JoinHorizontal(lipgloss.Center, WidgetContainer.Render(newContributorsDisplay), WidgetContainer.Render(alumniContributorsDisplay))
+
+	return contributorsMetrics
+}
+
+// drawDashboardView: this is the main model view (shows repository info and tables)
+func (m *DashboardModel) drawDashboardView() string {
+	if WindowSize.Width == 0 {
+		return "Loading..."
+	}
+	var wg sync.WaitGroup
+
+	wg.Add(3)
+	var titleView, repoInfoView, metricsView string
+	go func() {
+		defer wg.Done()
+		titleView = m.drawTitle()
+	}()
+
+	go func() {
+		defer wg.Done()
+		repoInfoView = m.drawRepositoryInfo()
+	}()
+
+	go func() {
+		defer wg.Done()
+		metricsView = m.drawMetrics()
+	}()
+	wg.Wait()
+
+	mainView := lipgloss.JoinVertical(lipgloss.Center, titleView, repoInfoView, metricsView, m.contributorErr)
+	_, h := lipgloss.Size(mainView)
+	if WindowSize.Height < h {
+		contentLeft := lipgloss.JoinVertical(lipgloss.Center, titleView, repoInfoView)
+		mainView = lipgloss.JoinHorizontal(lipgloss.Center, contentLeft, metricsView)
+	}
+	frame := Viewport.Render(mainView)
+	return frame
+}
+
+// validateShowQuery: validates fields set to query the contributor tables
+func validateShowQuery(opts *Options) error {
+	if opts.RepoName == "" {
+		return errors.New("no repository was provided")
+	}
+
+	if len(strings.Split(opts.RepoName, "/")) != 2 {
+		return errors.New("wrong repository name format. Must be in the form owner/name, i.e(open-sauced/pizza)")
+	}
```

Comment:
It'd be cool if we can _also_ accept `https://github.com/owner/repo` format and extract out the `owner` and `repo` since elsewhere in the CLI we accept that format (i.e., when using the `bake` command)

I hit this trying this with:
```
❯ pizza show https://github.com/open-sauced/api
Error: wrong repository name format. Must be in the form owner/name, i.e(open-sauced/pizza)
```